themes/default/content/blog/dotnet-custom-providers/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Preview of .NET Core resource providers"
+authors: ["fraser-waters"]
+tags: ["pulumi-news", ".net", "c#", "packages", "features", "native-providers"]
+meta_desc: "Today we are releasing preview support for writing Pulumi providers in any .NET Core language, including C#, F#, and VB.NET."
+date: "2023-01-18"
+
+meta_image: "pulumi-dotnet.png"
+---
+
+Today we are pleased to announce the Preview of .NET Core support for custom resource providers. This means you can build custom providers, using your favorite .NET language, including C#, F#, and VB.NET.
+
+<!--more-->
+
+This is a first look preview release of the provider interface for .NET, it has minimal documentation but is feature complete. We hope to gather early feedback to refine the API and quickly bring this to a production quality release.
```

Comment:
Maybe we need a conjunction after the ','.

> [..] of the provider interface for .NET, therefore it has [..]

But I guess that implies that "first look" = "minimal documentation."
I think a separate sentence might work better.

```suggestion
This is a first look preview release of the provider interface for .NET. It has minimal documentation but is feature complete. We hope to gather early feedback to refine the API and quickly bring this to a production quality release.
```